package.json

```diff
@@ -6,10 +6,13 @@
   "type": "module",
   "scripts": {
     "start": "npm run dev",
+    "start:test": "NODE_ENV=testing npm run dev",
     "lint": "eslint '**/*.js'",
     "lint:fix": "eslint '**/*.js' --fix",
     "build": "vite build",
-    "dev": "vite dev --host"
+    "dev": "vite dev --host",
+    "test": "backstop --config=backstop.cjs --docker test",
```

Comment:
Instead of calling backstop directly, maybe we call a script. That script may then also be able to verify if docker is installed / running?

Reply:
Yeah it would be better to inform users about possible problems. Actually that script can also install playwright browser and backstopjs (if not installed) also.

Reply:
With the latest changes, we don't use Docker so the test script just runs backstopjs.